frontend/src/App.tsx

```diff
@@ -13,14 +13,17 @@ import {
 import { IonReactRouter } from '@ionic/react-router'
 // import Amplify from 'aws-amplify'
 
-import { albums, person, image } from 'ionicons/icons'
+import { albums, person, image, barChart } from 'ionicons/icons'
 
 import { Redirect, Route } from 'react-router-dom'
 
 // import awsExports from './aws-exports'
-import Moments from './pages/moments'
-import Profile from './pages/profile'
-import Reflections from './pages/reflections'
+import {
+  MomentsView,
+  ProfileView,
+  ReflectionsView,
+  ProgressView,
```

Comment:
Does it make sense to change the naming to `*Pages`? E.g.:

* `MomentsPage`
* `ProfilePage`
* ...

Reply:
I would keep it since `pages` are the "stack" we have and each "page" could have multiple view, like `createNewMomentView`

Reply:
@DaemonLoki is View ok?

Reply:
Still generic but I'm fine with it.